test/integ/pandas/test_pandas_tools.py

```diff
@@ -47,13 +47,15 @@
 # Note: since the file will to small to chunk, this is only testing the put command's syntax
 @pytest.mark.parametrize("parallel", [4, 99])
 @pytest.mark.parametrize("quote_identifiers", [True, False])
+@pytest.mark.parametrize("auto_create_table", [True, False])
```

Comment:
do we also need to parametrize `create_temp_table` and test it?

Reply:
Yea its tested by in Snowpark, but I'll add a test here for this as well